src/components/Forms/ClientInfo.js

```diff
@@ -10,8 +11,75 @@ import MedicalHistory from "./MedicalHistory";
 import PhoneInput from "./PhoneInput";
 import { useTranslation } from "react-i18next";
 
+const genders = ["Male", "Female", "Not Specific"];
+
 export default function FormPropsTextFields() {
   const { t } = useTranslation();
+  const [firstName, setFirstName] = useState("");
+  const [middleName, setMiddleName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [gender, setGender] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [country, setCountry] = useState("");
+
+  const [birthDate, setBirthDate] = useState(Date.now());
+
+  const handleFirstNameChange = (event) => {
+    setFirstName(event.target.value);
+    // console.log(firstName);
+  };
+  const handleMiddleNameChange = (event) => {
+    setMiddleName(event.target.value);
+  };
+  const handleLastNameChange = (event) => {
+    setLastName(event.target.value);
+  };
+  const handleGenderChange = (event) => {
+    setGender(event.target.value);
+  };
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+  };
+  const handlePhoneChange = (event) => {
+    setPhone(event.target.value);
+  };
+  const handleAddressChange = (event) => {
+    setAddress(event.target.value);
+  };
+  const handleCityChange = (event) => {
+    setCity(event.target.value);
+  };
+  const handleCountryChange = (event) => {
+    setCountry(event.target.value);
+  };
+  const handleDateOfBirth = (date) => {
+    setBirthDate(date);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const formData = {
+      firstName,
+      middleName,
+      lastName,
+      gender,
+      email,
+      phone,
+      address,
+      city,
+      country,
+      birthDate,
+    };
+  };
+  React.useEffect(() => {
+    console.log("firstName", firstName);
```

Comment:
please remove the console. Production code shouldn't have console

Reply:
Will work on it later